packages/liburcu/liburcu.spec

```diff
@@ -0,0 +1,67 @@
+Name: %{_cross_os}liburcu
+Version: 0.14.0
+Release: 1%{?dist}
+Summary: Library for userspace RCU
+License: LGPL-2.1-or-later
+URL: http://liburcu.org
+Source0: http://lttng.org/files/urcu/userspace-rcu-latest-0.14.tar.bz2
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: Files for development using the library for userspace RCU
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n userspace-rcu-%{version} 
+
+%build
+%cross_configure 
+
+# "fix" rpath
+sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
+sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
+
+%make_build
+
+%install
+%make_install
+
+find %{buildroot} -type f -name "*.a" -delete
+
+%files
+%license LICENSE
```

Comment:
The sources distribute copies of the licenses that apply to the package, should you use those instead of the `LICENSE`file?